docs/index.md

```diff
@@ -0,0 +1,98 @@
+## Overview
+This guide walk-through the process of deploying an Sample Java application in CFT.
+
+This guide uses a pre-configured application from a template. Application is deployed to a kubernetes cluster. 
+At the end of this tutorial, you should be able to access an working application via VPN and made changes to it.
+
+## Prerequisite
+
+`Github` access is required to complete the steps in this tutorial. Following the [link](https://hmcts.github.io/onboarding/team/github.html#github) to request access.
+
+## High Level Steps
+
+- Create a Repository and build pipeline
+- Build application
+- Configure load balancing for HA
+- Deploy application
+- Access application
+- Customize application
+- Troubleshooting
+
+## Steps
+
+#### Create a Repository and build pipeline
```

Comment:
I think all fields should be specified for the Create a new component so the user doesn't get confused, e.g what github admin team should they use...